lib/fastlane/plugin/dynatrace/helper/dynatrace_helper.rb

```diff
@@ -114,6 +114,65 @@ def self.get_dss_client(params)
         return dtxDssClientPath
       end
 
+      def self.symlink_lldb(lldb_path, destination_path, auto_link = true)
+        require_path(destination_path)
+
+        unless lldb_path.nil?
+          return symlink(lldb_path, destination_path) if path_exists?(lldb_path)
+        end
+
+        if auto_link
+          return auto_link_lldb(destination_path)
+        end
+
+        # if there is no lldb_path provided and auto_link is not true
+        # then just delete the existing symlinks and raise error
+        Dir.glob("#{destination_path}/*LLDB*").map { |file| FileUtils.rm(file) if File.symlink?(file) }
```

Comment:
```suggestion
        Dir.glob("#{destination_path}/*LLDB.framework").map { |file| FileUtils.rm(file) if File.symlink?(file) }
```
Can it be more specific like this?